rclcpp_action/include/rclcpp_action/create_client.hpp

```diff
@@ -97,10 +97,10 @@ create_client(
  * \param[in] group The action client will be added to this callback group.
  *   If `nullptr`, then the action client is added to the default callback group.
  */
-template<typename ActionT>
+template<typename ActionT, typename NodeT = rclcpp::Node>
```

Comment:
I don't think the default value is necessary. Template argument deduction should suffice.
```suggestion
template<typename ActionT, typename NodeT>
```

Same for `create_server`.

Reply:
I'm not necessarily convinced that's true, but I can verify tomorrow